enarx-keep/tests/bin/read.c

```diff
@@ -0,0 +1,22 @@
+#include "libc.h"
+
+int main(void) {
+    int min(int a, int b) { return a < b ? a : b; }
+
+    char buf[12] = {'\0'};
+    int len = 1;
+    int bytes = 0;
+
+    while (1) {
+        bytes = read(STDIN_FILENO, buf, len);
+
+        if (bytes > 0) {
+           write(STDOUT_FILENO, buf, bytes);
+        } else
+            break;
```

Comment:
As a matter of good C practice, never unbalance your braces. It can lead to weird bugs. Do either this:

```suggestion
        if (bytes > 0)
            write(STDOUT_FILENO, buf, bytes);
        else
            break;
```

Or this (preferred by some):

```suggestion
        if (bytes > 0) {
            write(STDOUT_FILENO, buf, bytes);
        } else {
            break;
        }
```